timed/tracking/filters.py

```diff
@@ -134,9 +136,11 @@ def filter_editable(self, queryset, name, value):
                 return queryset.none()
 
             queryset = queryset.exclude(
-                Q(user__supervisors=user) |
-                Q(task__project__reviewers=user) |
-                Q(user=user)
+                (
+                    Q(user__supervisors=user) |
+                    Q(task__project__reviewers=user) |
+                    Q(user=user)
+                ) & Q(verified_by__isnull=True)
```

Comment:
Can we get rid of this code duplication?

Reply:
Agree. Didn't like this before either. Moved filter query into its own method.